shell/shell.go

```diff
@@ -337,6 +346,27 @@ func (sh *Shell) TranspileFile(in string, out string) error {
 	return errors.Join(err, werr)
 }
 
+// TranspileOnly only does direct transpiling of input to output
+// without adding anything to the resulting file.
+func (sh *Shell) TranspileOnly(in string, out string) error {
```

Comment:
This, similar to what I said about `Build` and `Transpile`, should be one function that either determines the distinction automatically or takes a bool flag for it. It seems like it would be very easy to detect if there is a `package main` line in the source code.